config/settings.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    syncTime: 60000 //one Minute
```

Comment:
separate 60000 to a constant:
const DEFAULT_SYNC_INTERVAL_IN_MILLISECONDS = 60000;

and change to use:
```
module.exports = {
   syncTime: process.env.SYNC_INTERVAL_IN_MILLISECONDS || DEFAULT_SYNC_INTERVAL_IN_MILLISECONDS
}
```

That way we can change the value through environment variables